velox/functions/lib/window/RegistrationFunctions.h

```diff
@@ -34,4 +34,24 @@ void registerRowNumberBigint(const std::string& name);
 // Register the Spark function row_number() with the integer data type
 // for the return value.
 void registerRowNumberInteger(const std::string& name);
+
+// Register the Presto function rank() with the bigint data type
+// for the return value.
+void registerRankBigint(const std::string& name);
+
+// Register the Spark function rank() with the integer data type
+// for the return value.
+void registerRankInteger(const std::string& name);
+
+// Register the Presto function dense_rank() with the bigint data type
+// for the return value.
+void registerDenseRankBigint(const std::string& name);
+
+// Register the Spark function dense_rank() with the integer data type
+// for the return value.
+void registerDenseRankInteger(const std::string& name);
+
+// Register the function percent_rank() both for Presto and Spark.
+void registerPercentRank(const std::string& name);
```

Comment:
Why do we need to move its registration to the lib? Is this function has the same semantics with Spark's? If so, can we add test for it and update the document?

Reply:
@rui-mo : This function is available in the common file itself so its registration is moved here.

Reply:
Got it.
> // Register the function percent_rank() both for Presto and Spark.

Maybe we can improve above comment, if we cannot ensure its semantics are the same with Spark.

Reply:
Updated. And the semantics for spark and presto is same.